python/paddle/fluid/tests/unittests/test_reshape_op.py

```diff
@@ -265,76 +273,107 @@ def test_1(self):
         assert np.array_equal(res_3, input.reshape([5, 10]))
         assert np.array_equal(res_4, input.reshape(shape))
 
+    def test_paddle_api(self):
+        self._set_paddle_api()
+        self._test_api()
+
+    def test_fluid_api(self):
+        self._set_fluid_api()
+        self._test_api()
+
+    def test_imperative(self):
```

Comment:
Should imperative test also uses self.fill_constant, self.data, self.reshape to match your other test style?

Reply:
Thanks, done